### PR TITLE
export offline methods to top-level of package

### PR DESCRIPTION
**Pull request by khancode**
## High Level Overview of Change

- All methods that don’t require a network connection are exported at the top-level of the package.
- These instance methods are now marked as deprecated.
- Removed an empty ts file

### Context of Change

[jira ticket](https://ripplelabs.atlassian.net/browse/DEVX-81?atlOrigin=eyJpIjoiNjZiMjNlNWM4MWY4NDUzZGFkNzA3ODMxYWIxYmYwMjciLCJwIjoiaiJ9)

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Test Plan

CI Passes

### CODE REVIEW

**Review by intelliot — APPROVED** [review]
LGTM

**Review by natenichols — APPROVED** [review]
Well Done! LGTM 👍

File: src/offline/utils.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import signPaymentChannelClaim from './sign-payment-channel-claim'
+import verifyPaymentChannelClaim from './verify-payment-channel-claim'
+import { dropsToXrp,
+    xrpToDrops,
+    toRippledAmount,
+    convertKeysFromSnakeCaseToCamelCase,
+    removeUndefined,
+    rippleTimeToISO8601,
+    iso8601ToRippleTime,
+    isValidSecret } from '../common/utils'
```
Comment (ledhed2222) [inline src/offline/utils.ts:12]: add trailling comma and line break

File: src/offline/utils.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    computeLedgerHash,
+    computeEscrowHash,
+    computePaymentChannelHash,
+} from '../common/hashes'
+import { generateAddressAPI, GenerateAddressOptions, GeneratedAddress } from '../offline/generate-address'
+
+// @deprecated Use X-addresses instead
+const generateAddress = (options: GenerateAddressOptions = {}): GeneratedAddress => {
+    return generateAddressAPI({...options, includeClassicAddress: true})
+}
```
Comment (ledhed2222) [inline src/offline/utils.ts:32]: could be rewritten in simpler style:

```javascript
const generateAddress = (options: GenerateAddressOptions = {}): GeneratedAddress => (
  generateAddressAPI({...options, includeClassicAddress: true})
)
```

File: src/offline/utils.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+const generateXAddress = (options: GenerateAddressOptions = {}): GeneratedAddress => {
+    return generateAddressAPI(options)
+}
```
Comment (ledhed2222) [inline src/offline/utils.ts:36]: ditto

Comment (ledhed2222) [reply src/offline/utils.ts:36]: actually, even simpler:

```javascript
const generateXAddress = generateAddressAPI
```

**Review by ledhed2222 — CHANGES_REQUESTED** [review]
some nitpicky comments, but looks great!

File: src/offline/utils.ts (hunk 3 above)
Comment (khancode) [reply src/offline/utils.ts:36]: Ah right, since they have the same parameters. I'll make this change, thanks!

File: src/offline/utils.ts (hunk 2 above)
Comment (khancode) [reply src/offline/utils.ts:32]: I agree, I'll do this.

File: src/offline/utils.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    computeSignerListLedgerObjectID,
+    computeOrderID,
+    computeTrustlineHash,
+    computeTransactionTreeHash,
+    computeStateTreeHash,
+    computeLedgerHash,
+    computeEscrowHash,
+    computePaymentChannelHash,
+    generateAddress,
+    generateXAddress,
```
Comment (ledhed2222) [inline src/offline/utils.ts:60]: actually - you can remove line 35 altogether:

```javascript
generateAddressAPI as generateXAddress,
```

Comment (ledhed2222) [reply src/offline/utils.ts:60]: alternatively, you can import it that way and leave the export alone. your call

Comment (khancode) [reply src/offline/utils.ts:60]: That's so convenient! haha

**Review by ledhed2222 — APPROVED** [review]

**Pull request merged**